stub/src/main/java/io/grpc/stub/AbstractStub.java

```diff
@@ -149,6 +151,10 @@ public final S withDeadlineAfter(long duration, TimeUnit unit) {
     return build(channel, callOptions.withDeadlineAfter(duration, unit));
   }
 
+  public final S withDeadlineAfter(Duration duration) {
```

Comment:
Missing unit test.

Reply:
Included required unit test. Requesting you to check.